po/pt-br.po

```diff
@@ -0,0 +1,163 @@
+msgid "Quit"
+msgstr "Fechar"
+
+#. translators: This is a menu entry.
+#: src/app/components/user_menu/user_menu.rs:67
+msgid "Log out"
+msgstr "Sair"
+
+#. translators: This is a sidebar entry to browse to saved albums.
+#: src/app/components/navigation/home.rs:33
+msgid "Library"
+msgstr "Biblioteca"
+
+#. translators: This is a sidebar entry to browse to saved playlists.
+#: src/app/components/navigation/home.rs:38
+msgid "Playlists"
+msgstr "Listas de reprodução"
+
+#. This is the visible name for the play queue. It appears in the sidebar as well.
+#: src/app/components/navigation/home.rs:43
+#: src/app/components/now_playing/now_playing.ui:21
+msgid "Now playing"
+msgstr "Tocando agora"
+
+#. translators: This text is part of a larger text that says "Search results for <search term>".
+#: src/app/components/search/search.rs:123
+msgid "Search results for"
+msgstr "Procurar por"
+
+#. translators: This is part of a contextual menu attached to a single track; this entry allows viewing the album containing a specific track.
+#: src/app/components/labels.rs:5
+msgid "View album"
+msgstr "Ver álbum"
+
+#. translators: This is part of a contextual menu attached to a single track; the full text is "More from <artist>".
+#: src/app/components/labels.rs:8
+msgid "More from"
+msgstr "Masi de"
+
+#. translators: This is part of a contextual menu attached to a single track; the intent is to copy the link (public URL) to a specific track.
+#: src/app/components/labels.rs:11
+msgid "Copy link"
+msgstr "Copiar link"
+
+#. translators: This is part of a contextual menu attached to a single track; this entry adds a track at the end of the play queue.
+#: src/app/components/labels.rs:14
+msgid "Add to queue"
+msgstr "Adicionar a fila"
+
+#. translators: This is part of a contextual menu attached to a single track; this entry removes a track from the play queue.
+#: src/app/components/labels.rs:17
+msgid "Remove from queue"
+msgstr "Remover da fila"
+
+#. translators: This is part of a larger text that says "Add to <playlist name>". This text should be as short as possible.
+#: src/app/components/labels.rs:24
+msgid "Add to {}"
+msgstr "Adicionar a {}"
+
+#. translators: This notification is the default message for unhandled errors. Logs refer to console output.
+#: src/app/components/mod.rs:100
+msgid "An error occured. Check logs for details!"
+msgstr "Ocorreu um erro. Verifique os logs para mais detalhes!"
+
+#. This text appears when entering selection mode. It should be as short as possible.
+#: src/app/components/selection/selection_heading.rs:74 src/window.ui.in:26
+msgid "No song selected"
+msgstr "Nenhuma música selecionada"
+
+#. translators: This is part of a larger text that says "<n> songs selected" when in selection mode. This text should be as short as possible.
+#: src/app/components/selection/selection_heading.rs:88
+msgid "song selected"
+msgid_plural "songs selected"
+msgstr[0] "música selecionada"
+msgstr[1] "Músicas selecionadas"
```

Comment:
absolutely :) 

> This is part of a larger text that says "<n> songs selected" when in selection mode. This text should be as short as possible.

I should migrate this kind of label so that they work like msgid="Add to {}"